docs/administration/back_office/back_office_tabs/back_office_tabs.md

```diff
@@ -71,7 +71,7 @@ To create a tab group, register it as a service:
 [[= include_file('code_samples/back_office/dashboard/article_tab/config/custom_services.yaml', 0, 1) =]][[= include_file('code_samples/back_office/dashboard/article_tab/config/custom_services.yaml', 7, 13) =]]
 ```
 
-Tag the group with `ibexa.admin_ui.component`.
+Tag the group with `ibexa.twig.component`.
 `group` indicates where the group is rendered.
 
-For a list of possible rendering places, see [Injecting custom components](custom_components.md).
+To learn more about this mechanism and the available groups, see [Twig Components](components.md).
```

Comment:
As we are in BO context, I would directly link to [BO groups table](https://ez-systems-developer-documentation--2739.com.readthedocs.build/en/2739/administration/back_office/back_office_elements/custom_components/).

Maybe both.

```suggestion
To learn more about this mechanism, see [Twig Components](components.md).
And for the groups available in the back office, see [Back office elements > Custom components](custom_components.md).
```